names/jlokhande46.md

```diff
@@ -0,0 +1,4 @@
+- [jlokhande46](https://github.com/jlokhande46)
+...- www.jayeshlokhande.me
+...- https://twitter.com/jlokhande940
+...- https://www.linkedin.com/in/jayesh-lokhande-3a6315111/
```

Comment:
Hi @jlokhande46 the three dots are placeholder for three spaces needed for indentation.

Please add the change and push your changes to the same branch again :)